.github/workflows/autocommit.yml

```
@@ -0,0 +1,21 @@
+name: Commit to keep branch open
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  check-commits:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
```

Use the latest major version of the action.

```suggestion
      uses: actions/checkout@v4
```